samples/apps/copilot-chat-app/webapi/CopilotChat/Models/DocumentImportForm.cs

```diff
@@ -42,4 +42,9 @@ public enum DocumentScopes
     /// Will be use to validate if the user has access to the chat session.
     /// </summary>
     public string UserId { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Name of the user who sent this file.
+    /// </summary>
+    public string UserName { get; set; } = string.Empty;
```

Comment:
To my understanding, UserName is PII. The team might have concerns about storing this info without proper data handling policy implemented. 
A potential way to workaround this is to call Graph API to get user info. However, if the user is an MSA account, we haven't found a way to get user infor from MSA identity provider. You can talk to @TaoChenOSU to get his learnings from multi-user chat.

Reply:
Correct - we should not be storing usernames or email addresses directly.

Reply:
I recommend letting this be a runtime translation from the userId through the security provider, such as AAD. We don't want to store identifiable information if we can help it.

Reply:
yes, this actually came up in the multi-user chat discussion earlier today. I decided to go this route since we are already doing the [same for ChatMessage](https://github.com/microsoft/semantic-kernel/blob/1a49468f5306060cf821d24f3a594d4819e86111/samples/apps/copilot-chat-app/webapi/CopilotChat/Models/ChatMessage.cs#LL53C5-L53C41). 
@TaoChenOSU is currently showing GUIDs for the user's name and is considering removing the column for now I believe.

I think that if we're not comfortable with saving the username here, which makes sense since we want to move away from this pattern, I can remove the column as well.

Reply:
this is what it looks like without the username, just for now at least:
<img width="1508" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/52973358/699d9795-2eca-4d2d-b181-192aeaeff0c9">

after this PR I can take a look at fetching using the user's ID if no one is doing so yet

Reply:
Good call